src/bonsai/websocket/markets.ts

```
@@ -110,7 +110,54 @@ export function setUpMarkets(store: RootStore) {
+            // Base market data
+            ...market,
+
+            // Derived display items
+            assetId: market.ticker.split('-')[0], // Extract asset from ticker like "MIRG.BA-USD" -> "MIRG.BA"
+            displayableAsset: market.ticker.split('-')[0],
+            displayableTicker: market.ticker,
+
+            // Derived market core
+            effectiveInitialMarginFraction: parseFloat(market.initialMarginFraction) || null,
+            openInterestUSDC: parseFloat(market.openInterest) * parseFloat(market.oraclePrice || '0'),
+            percentChange24h: parseFloat(market.priceChange24H) || null,
```

ESLint

[error] Insert `··`

(prettier/prettier)